charts/app-config/values-integration.yaml

```diff
@@ -553,6 +553,21 @@ govukApplications:
         value: "https://licensify.integration.govuk-internal.digital"
       - name: BACKEND_URL_search-api
         value: "https://search-api.integration.govuk-internal.digital"
+- name: draft-service-manual-frontend
```

Comment:
nit: could you move this underneath `service-manual-frontend` please so that it's easier to see the differences?

Reply:
thought we was ordering alphabetically? - but yes that would be beneficial - happy to do that if it makes it easier